calico_test/tests/st/utils/utils.py

```diff
@@ -107,19 +111,21 @@ def debug_failures(fn):
     :param fn: The function to decorate.
     :return: The decorated function.
     """
+
     def wrapped(*args, **kwargs):
         try:
             return fn(*args, **kwargs)
         except KeyboardInterrupt:
             raise
         except Exception as e:
-            if (os.getenv("DEBUG_FAILURES") != None and
-                os.getenv("DEBUG_FAILURES").lower() == "true"):
+            if (os.getenv("DEBUG_FAILURES") is not None and os.getenv(
+                    "DEBUG_FAILURES").lower() == "true"):
                 logger.error("TEST FAILED:\n%s\nEntering DEBUG mode."
                              % e.message)
                 pdb.set_trace()
             else:
                 raise e
+
     return wrapped
 
 
```

Comment:
check_bird_status()

This needs updating to run `calicoctl node status` rather than `calicoctl status`

Do we have any tests for that?